doc/installing.rst

```diff
@@ -6,11 +6,160 @@ Installing PYPIT
 
 This document will describe how to install PYPIT.
 
-Source Code
-===========
+Installing Dependencies
+=======================
+Though we have tried to keep the number of dependencies low, there are a few packages that need to be installed (various python packages, GSL, and linetools).
 
-Path
-====
+In general, we recommend that you use Anaconda for the majority of these installations. 
 
-Cython
-======
+Detailed installation instructions are presented below:
+
+Python Dependencies
+-------------------
+
+PYPIT depends on the following list of Python packages. 
+
+We recommend that you use `Anaconda <https://www.continuum.io/downloads/>`_ to install and/or update these packages.
+
+* `python <http://www.python.org/>`_ versions 2.7, or 3.3 or later
+* `numpy <http://www.numpy.org/>`_ version 1.10 or later
+* `astropy <http://www.astropy.org/>`_ version 1.0 or later
+* `scipy <http://www.scipy.org/>`_ version 0.17 or later
+* `matplotlib <http://matplotlib.org/>`_  version 1.4 or later
+* `PyQT4 <https://wiki.python.org/moin/PyQt/>`_ version 4 (needed for linetools)
+
+If you are using Anaconda, you can check the presence of these packages with::
+
+	conda list "^python$|numpy|astropy$|scipy$|matplotlib|PyQT"
+
+If the packages have been installed, this command should print out all the packages and their version numbers.  
+
+If any of these packages are missing you can install them with a command like::
+
+	conda install PyQT
+
+If any of the packages are out of date, they can be updated with a command like::
+
+	conda update scipy
+
+Installing Linetools
+--------------------
+The latest version of `Linetools <https://github.com/linetools/linetools/>`_ is also required for PYPIT. 
+Linetools is a package designed for the analysis of 1-D spectra. The installation steps for linetools are provided below but much more detailed documentation is provided `here <http://linetools.readthedocs.io/en/latest/install.html/>`_. 
+
+In addition to the python packages mentioned above, linetools requires the installation of `specutils <https://github.com/astropy/specutils/>`_. Specutils can either be cloned directly from github or installed via pip.
```

Comment:
specutils is no longer required with linetools (quite the opposite).

I'd reduce this section about linetools.  Maybe even remove it altogether.  It is no more difficult to install than scipy.  You can
even pip install it.